pkg/apis/agones/v1/gameserver.go

```diff
@@ -311,6 +304,18 @@ type PlayerStatus struct {
 	IDs      []string `json:"ids"`
 }
 
+// CounterStatus stores the current counter values
+type CounterStatus struct {
```

Comment:
Since it's used in both Spec and Status, maybe the names should be `Counter` and `List` ? WDYT?